### ISSUE TITLE
the models are not displayed

### ISSUE THREAD

**Issue by faselunare** [wintercms/wn-builder-plugin#3]
Winter version: Detected a modified version of Winter CMS build 1.1.2
PHP: 8.0.3

Step to reproduce
- create new plugin
- create new model(s)
- the models are not displayed under the models tabs but the file exists under the folder Models

**Comment by LukeTowers** [comment]
@faselunare are you sure that the correct plugin is selected?

**Comment by faselunare** [comment]
hi @LukeTowers i have only one plugin at the moment
I have never had this type of problem, today I try to do a new winter installation from scratch

**Comment by faselunare** [comment]
I created a fresh install, only installed the builder but the problem is the same.
I'm on macosx with Valet 2.14.1

**Comment by LukeTowers** [comment]
Details from @mjauvin below. Looks like it's only happening on PHP 8.

this is the code that fails:
```php
    protected function extractNamespace($stream)
    {   
        if ($stream->getNextExpected(T_WHITESPACE) === null) {
            return null;
        }

        return $stream->getNextExpectedTerminated([T_STRING, T_NS_SEPARATOR], [T_WHITESPACE, ';']);
    }
```
`getNextExpectedTerminated()` seems to fail now for the namespace on PHP 8

**Comment by mjauvin** [comment]
Fixed by 98dc18d0b62cccef3f52b7987160c3596180eaeb
